components/admin_console/admin_definition.jsx

```diff
@@ -3950,6 +3989,7 @@ const AdminDefinition = {
                         type: Constants.SettingsTypes.TYPE_CUSTOM,
                         component: SAMLFeatureDiscovery,
                         key: 'SAMLFeatureDiscovery',
+                        isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.AUTHENTICATION.SAML)),
```

Comment:
Allows to create a trial license. Will require write permisson on ABOUT_ EDITION_AND_LICENSE